keycloak-rest-api-extensions/src/main/java/io/cloudtrust/keycloak/services/resource/api/account/FixedAccountRestService.java

```diff
@@ -86,4 +100,53 @@ public Response deleteAccount() {
         }
     }
 
+    /**
+     * Send execute actions email
+     * @param lifespan
+     * @param actions
+     * @return
+     */
+    @Path("execute-actions-email")
+    @PUT
+    @Consumes(MediaType.APPLICATION_JSON)
+    public Response executeActionsEmail(@QueryParam("lifespan") Integer lifespan, List<String> actions) {
+        if (user.getEmail()==null) {
+            return ErrorResponse.error("User email missing", Status.BAD_REQUEST);
+        }
+
+        if (!user.isEnabled()) {
+            throw new WebApplicationException(
+                ErrorResponse.error("User is disabled", Status.BAD_REQUEST));
+        }
+
+        if (lifespan == null) {
+            lifespan = realm.getActionTokenGeneratedByAdminLifespan();
+        }
+        int expiration = Time.currentTime() + lifespan;
+        String redirectUri = null;
+        String clientId = Constants.ACCOUNT_MANAGEMENT_CLIENT_ID;
```

Comment:
I don't understand where does this Constants come from ... I don't see it in imports neither in package. What am I missing ?

Reply:
It comes from org.keycloak.models
(the code to send the mail is mainly copied from Keycloak)